src/components/navbar/index.cjsx

```diff
@@ -53,6 +54,9 @@ module.exports = React.createClass
   transitionToMenuItem: (routeName, params) ->
     @context.router.transitionTo(routeName, params)
 
+  shouldHide: ->
+    @state.scrollTop > 60 and document.getElementsByClassName('task')
```

Comment:
Also, @Fredasaurus Would it be OK to always hide the top nav bar when scrolling? (and have it "pop" back in when scrolling up?)